blocks/product-content-block/index.story.jsx

```diff
@@ -24,7 +24,7 @@ const mockData = {
 };
 
 export const TitleAndContent = () => (
-	<ProductContentDisplay summary="Product Details">
+	<ProductContentDisplay summary={mockData.schema.productDetails.label}>
 		{mockData.schema.productDetails.value}
 	</ProductContentDisplay>
 );
```

Comment:
Do we want additional stories for cases like:
1. Using a custom title?
2. Displaying the details with a content initially expended?
Thanks!